packages/connect/src/types/device.ts

```diff
@@ -51,6 +60,11 @@ export type KnownDevice = {
     features: PROTO.Features;
     unavailableCapabilities: UnavailableCapabilities;
     availableTranslations: string[];
+    authenticityChecks: {
+        firmwareRevision: FirmwareRevisionCheckResult;
+        // Maybe add FirmwareHashCheck result here?
+        // Maybe add AuthenticityCheck result here?
+    };
```

Comment:
yes, it makes sense to me. define logic for firmwareHashCheck and authenticityCheck similarly to firmwareRevision check so that it is available also internally in connect(not only on API), and save its result value with device object. 

searching for volunteers 🧑‍🏭